docs/UserGuide.md

```diff
@@ -108,19 +115,19 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com i/S1234123A a/John Street, Block 123, #01-01 t/2`
 * `add n/Betsy Crowe p/24681357 e/betsycrowe@example.com i/T1234567D a/Clementi Ave 1, Block 230 t/1 tag/Diabetic tag/G6PD`
 
+> **Note:** `add` **does not support** the addition of appointments. Use the [appointment](#adding-an-appointment--appointment) command to do so.
+
 ### Adding a remark to a patient profile : `remark`
 
 Adds a remark to a specified patient profile.
-> 💡 **Note:** You can only specify **one** remark to add
+> **Note:** You can only specify **one** remark to add, and patients can only have one remark.
```

Comment:
maybe can add on that if remark command is executed by the user for a patient that already has a remark, the original will be overwritten